atmo/clusters/views.py

```diff
@@ -25,10 +31,10 @@ def new_cluster(request):
             )
         )
         return redirect('keys-new')
-    initial = {
-        'identifier': '{}-telemetry-analysis'.format(user_display(request.user)),
-        'size': 1,
-    }
+    elif ssh_key_count == 1:
+        # If only 1 ssh key, make it pre-selected.
+        initial['ssh_key'] = request.user.created_sshkeys.values('pk')[0]['pk']
```

Comment:
Another option that's a simple single line change from what's there now is: `request.user.created_sshkeys.get().pk` since we know there's only 1 already.